src/macaron/slsa_analyzer/checks/build_as_code_check.py

```diff
@@ -105,7 +123,7 @@ def run_check(self, ctx: AnalyzeContext, check_result: CheckResult) -> CheckResu
         """
         # Get the build tool identified by the mcn_version_control_system_1, which we depend on.
         build_tool = ctx.dynamic_data["build_spec"].get("tool")
-        ci_services = ctx.dynamic_data["ci_services"]
+        ci_services = ctx.dynamic_data["ci_services"]  # keep storing things here
```

Comment:
Instead of `# keep storing things here` can you please add a more descriptive comment? Also always end a sentence with `.` :smile:

Reply:
This was a comment that I meant to remove :sleepy:, my bad.

Reply:
Fixed in [9c167f2](https://github.com/oracle-samples/macaron/pull/67/commits/9c167f271c3b880c3db95d6de5fc7bb51bdfcf3d).